acestep/audio_utils.py

```
@@ -181,18 +197,31 @@ def _save_mp3(
+            elif isinstance(e, subprocess.TimeoutExpired):
+                reason = "ffmpeg MP3 export timed out after 120 seconds."
+            else:
+                stderr = e.stderr.decode('utf-8', errors='ignore') if e.stderr else str(e)
+                reason = f"ffmpeg MP3 export failed: {stderr}"
+
+            # The WAV was already synthesized successfully before the ffmpeg
+            # step -- preserve it instead of discarding a valid result.
+            wav_fallback_path = output_path.with_suffix(".wav")
+            try:
+                shutil.move(str(temp_wav_path), str(wav_fallback_path))
+            except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)